service/src/test/java/tds/assessment/repositories/AssessmentQueryRepositoryImplIntegrationTests.java

```diff
@@ -134,9 +199,94 @@ public void shouldFindSegmentedAssessmentByKey() {
         assertThat(segment2.getKey()).isEqualTo("(SBAC_PT)SBAC-SEG2-MATH-8-Spring-2013-2015");
         assertThat(segment2.getSegmentId()).isEqualTo("SBAC-SEG2-MATH-8");
         assertThat(segment2.getSelectionAlgorithm()).isEqualTo("fixedform");
+        assertThat(segment2.getPosition()).isEqualTo(2);
+        assertThat(segment2.getMinItems()).isEqualTo(4);
+        assertThat(segment2.getMaxItems()).isEqualTo(4);
         assertThat(segment2.getSubject()).isEqualTo(subject);
         assertThat(segment2.getLanguages()).hasSize(2);
         assertThat(segment2.getLanguages()).contains(new Property("Language", "ENU"), new Property("Language", "Braille-ENU"));
         assertThat(segment2.getStartAbility()).isEqualTo(0);
+
+        List<Form> formsSeg1 = segment1.getForms();
+        assertThat(formsSeg1.size()).isEqualTo(3);
+        List<Form> formsSeg2 = segment2.getForms();
+        assertThat(formsSeg2.size()).isEqualTo(3);
+
+        List<Form> allSegments = new ArrayList<>();
+        allSegments.addAll(formsSeg1);
+        allSegments.addAll(formsSeg2);
+        Form form1 = null;
+        Form form2 = null;
+        Form form3 = null;
+        Form form4 = null;
+        Form form5 = null;
+        Form form6 = null;
+
+        for(Form form : allSegments) {
```

Comment:
Same question here.  Are the six forms here different resulting in different logic branches in the query being exercised?  Looking at the query the forms are a simple join so I don't know why we need all these different records.